flake.nix

```diff
@@ -1,34 +1,13 @@
 {
   description = "Repository of shared modules";
+  outputs = _: {
+    flakeModules = {
+      deploy-rs = import ./flake-modules/deploy-rs;
+      lynx-docs = import ./flake-modules/lynx-docs;
+    };
 
-  inputs = {
-    nixpkgs.url = "github:nixos/nixpkgs";
-    parts.url = "github:hercules-ci/flake-parts";
+    nixosModules = {
+      globals = import ./nixos-modules/globals.nix;
+    };
   };
-
-  outputs = inputs@{self, parts, ...}:
-    parts.lib.mkFlake { inherit inputs; }
-    ({ withSystem, flake-parts-lib, ... }:
-    let
-      inherit (flake-parts-lib) importApply;
-      flakeModules = {
-        deploy-rs = importApply ./flake-modules/deploy-rs { inherit withSystem; };
-        lynx-docs = importApply ./flake-modules/lynx-docs { inherit withSystem; };
-      };
-    in
-    {
-      systems = ["x86_64-linux"];
-      imports = [flakeModules.lynx-docs];
-
-      flake = {
-        inherit flakeModules;
-        repository = {
-          flake = "github:the-computer-club/lynx/";
-          uri = "https://github.com/the-computer-club/lynx/tree/main/";
-        };
-        nixosModules = {
-          globals = import ./nixos-modules/globals.nix;
-        };
-      };
-    });
```

Comment:
we no longer construct any `pkgs` variable, or do not expect any inputs.

the end-user's flake imports nixpkgs, and we'll use whatever instance they're using.